packages/compatibility/backend/sirius-components-compatibility/src/main/java/org/eclipse/sirius/components/compatibility/services/diagrams/CompatibilityReconnectionToolsExecutor.java

```diff
@@ -184,16 +184,18 @@ private boolean canReconnect(VariableManager variableManager, ReconnectEdgeDescr
         return canReconnect;
     }
 
-    private VariableManager createVariableManager(ReconnectionToolInterpreterData toolInterpreterData) {
+    private VariableManager createVariableManager(ReconnectionToolInterpreterData toolInterpreterData, IEditingContext editingContext) {
         VariableManager variableManager = new VariableManager();
         variableManager.put(IInterpreterSiriusVariables.DIAGRAM, toolInterpreterData.getDiagramContext().getDiagram());
         variableManager.put(IInterpreterSiriusVariables.SOURCE, toolInterpreterData.getSemanticReconnectionSource());
         variableManager.put(IInterpreterSiriusVariables.SOURCE_VIEW, toolInterpreterData.getReconnectionSourceView());
         variableManager.put(IInterpreterSiriusVariables.TARGET, toolInterpreterData.getSemanticReconnectionTarget());
         variableManager.put(IInterpreterSiriusVariables.TARGET_VIEW, toolInterpreterData.getReconnectionTargetView());
         variableManager.put(IInterpreterSiriusVariables.ELEMENT, toolInterpreterData.getSemanticElement());
-        variableManager.put("otherEnd", toolInterpreterData.getOtherEdgeEnd()); //$NON-NLS-1$
+        variableManager.put("semanticOtherEnd", toolInterpreterData.getOtherEdgeEnd()); //$NON-NLS-1$
+        variableManager.put("otherEnd", toolInterpreterData.getSemanticOtherEdgeEnd()); //$NON-NLS-1$
         variableManager.put("edgeView", toolInterpreterData.getEdgeView()); //$NON-NLS-1$
```

Comment:
All these names should probably be constants defined in `ReconnectionToolInterpreterData`.

Reply:
Should we consider the variable name change from `otherEnd` to `semanticOtherEnd` an API break to mention in the changelog?

Reply:
I have discussed that with SBE and he was not sure where to put them. He told me that anyway the variables names were more stable than the constants and their locations.